apache/spark
# [SPARK-43147] fix flake8 lint for local check

### What changes were proposed in this pull request?

Change flake8 config file

### Why are the changes needed?

Before changing this, when doing local python lint `./dev/lint-python`, it checks folder that shouldn't be checked. This doesn't happen in github action.

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Manually tested
Before: 
```
starting flake8 test...
......many more......
./python/docs/source/conf.py:112:1: E265 block comment should start with '# '
#source_encoding = 'utf-8-sig'
^
./python/docs/source/conf.py:132:1: E265 block comment should start with '# '
#language = None
^

./python/venv/lib/python3.8/site-packages/pkg_resources/_vendor/more_itertools/more.py:1814:13: E731 do not assign a lambda expression, use a def
            key_argument = lambda zipped_items: key(
            ^
./dev/ansible-for-test-node/roles/jenkins-worker/files/util_scripts/post_github_pr_comment.py:7:1: F401 'urllib.parse' imported but unused
import urllib.parse
^
./dev/ansible-for-test-node/roles/jenkins-worker/files/util_scripts/session_lock_resource.py:130:9: F841 local variable 'f' is assigned to but never used
        f = _acquire_lock(lock_filename, timeout_secs, lock_message)

......many more......
```
And many more

After:
```
starting flake8 test...
flake8 checks passed.
```
